### PR TITLE
bird_species_classification

### PR DESCRIPTION
**Pull request by IkkiOcean**
# Pull Request for PyVerse 💡

Kindly review the code and models. Feedback and suggestions are welcome.
## Issue Title : Addition of Deep Learning Models for Bird Species Classification

- **Info about the related issue (Aim of the project)** : This project aims to develop an automated solution using deep learning to accurately identify bird species from images, helping in research, conservation efforts, and birdwatching communities.
- **Name:** Vivek Prakash
- **GitHub ID:** https://github.com/IkkiOcean
- **Email ID:** vivekprakash.st@gmail.com
- **Idenitfy yourself:**  GSSOC-extd and hacktoberfest

Closes: #60 

### Describe the add-ons or changes you've made 📃

The models were fine-tuned using the dataset to achieve optimal performance for bird species classification. Pretrained ImageNet weights were used as a baseline, followed by specific training on the bird species images.

Implemented Models:

The following advanced deep learning architectures have been applied for the classification task:

	•	InceptionResNetV2
	•	InceptionV3
	•	VGG16

## Type of change ☑️

- [x] My code follows the code style of this project.
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

- [x] My code follows the code style of this project.
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by UTSAVS26 — DISMISSED** [review]
please add readme file of this [format](https://github.com/UTSAVS26/PyVerse/blob/main/.github%2Freadme_template.md)

**Review by UTSAVS26 — APPROVED** [review]

**Comment by IkkiOcean** [comment]
@UTSAVS26 it says the repository is not participating in hacktoberfest. You either have to add hacktoberfest tag in the repository  or hacktoberfest-accepted tag after merging the PR.

**Comment by UTSAVS26** [comment]
> @UTSAVS26 it says the repository is not participating in hacktoberfest. You either have to add hacktoberfest tag in the repository  or hacktoberfest-accepted tag after merging the PR.
> Screenshot 2024-10-04 at 2 55 53 PM
> 

Sorry I didn't update the labels before it is set to `request changes`
